bin/mm-format-ns-profile.sh

```diff
@@ -2,18 +2,32 @@
 
 # Author: Ben West
 
+self=$(basename $0)
 SETTINGS=${1-monitor/settings.json}
 CARBS=${2-monitor/carb-ratios.json}
 BASALRATES=${3-monitor/active-basal-profile.json}
 SENSITIVITIES=${4-monitor/insulin-sensitivities.json}
 TARGETS=${5-monitor/bg-targets.json}
-OUTPUT=${6-settings/ns-profile.json}
+OUTPUT=${6-/dev/fd/1}
```

Comment:
Where to `cat` the results (`-` aka, stdout is also named `/dev/fd/1`)  Many tools allow for an `--output -` or any other file, this is the bashism to do that.